cluster/local/integration_tests.sh

```diff
@@ -71,7 +71,7 @@ docker tag "${BUILD_IMAGE}" "${CROSSPLANE_IMAGE}"
 "${KIND}" load docker-image "${CROSSPLANE_IMAGE}" --name="${K8S_CLUSTER}"
 
 echo_step "installing helm package(s) into \"${CROSSPLANE_NAMESPACE}\" namespace"
-"${HELM3}" install --create-namespace -n "${CROSSPLANE_NAMESPACE}" "${PROJECT_NAME}" "${projectdir}/cluster/charts/${PROJECT_NAME}" --set replicas=2,args={'-d'},rbacManager.replicas=2,rbacManager.args={'-d'},image.pullPolicy=Never,imagePullSecrets=''
+"${HELM3}" install --create-namespace -n "${CROSSPLANE_NAMESPACE}" "${PROJECT_NAME}" "${projectdir}/cluster/charts/${PROJECT_NAME}" --set replicas=2,args={'-d'},rbacManager.replicas=2,rbacManager.args={'-d'},image.pullPolicy=Never,imagePullSecrets='',image.tag=${helm_tag}
```

Comment:
Do we need to set the image tag explicitly here if we are falling back to chart version as the default?

Reply:
I was hoping the same here, but turns out we are not populating the helm chart's `Chart.yaml' before running the e2es, so it won't be populated with a meaningful version and then I think we would want to set the same tag we just used to upload images nonetheless.

Reply:
Ah, I didn't notice we were installing from the `cluster/` dir rather than the built `_output`. Since we have to build the image anyway, we could opt to install from `_output`, but the nice aspect of the current approach is that we only have to rebuild the image, not the helm chart as well 👍🏻